.travis.yml

```diff
@@ -8,3 +8,9 @@ env:
   - STACK=cedar-14
 script:
   - bin/docker-build.sh $STACK
+  - |
```

Comment:
@edmorley, unfortunately, some of those packages are used by [heroku-buildpack-google-chrome](https://github.com/heroku/heroku-buildpack-google-chrome) and while it's unlikely that it's being used at runtime, it's an unnecessary risk.